<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WhatsNewWidget_0_8</class>
 <widget class="QWidget" name="WhatsNewWidget_0_8">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>616</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true" extracomment="not translatable because not shown to the user">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="bottomMargin">
         <number>16</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="QWidget" name="featureBoxes" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>79</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>16</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>16</number>
           </property>
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QWidget" name="inboxBox" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-width: 2px;
border-style: solid;
border-radius: 4px;
border-color: white;
color: white;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="margin">
               <number>8</number>
              </property>
              <item alignment="Qt::AlignTop">
               <widget class="QLabel" name="inboxBoxHeader">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-style: none;</string>
                </property>
                <property name="text">
                 <string>Inbox</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="inboxBoxImage">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-style:none;
margin-bottom: 4px;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QWidget" name="urlLookupBox" native="true">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-width: 2px;
border-style: solid;
border-radius: 4px;
border-color: grey;
color: grey;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="margin">
               <number>8</number>
              </property>
              <item alignment="Qt::AlignTop">
               <widget class="QLabel" name="urlLookupBoxHeader">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border:none</string>
                </property>
                <property name="text">
                 <string>Open URL</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="urlLookupBoxImage">
                <property name="minimumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="trendingBox" native="true">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>79</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-width: 2px;
border-style: solid;
border-radius: 4px;
border-color: grey;
color: grey;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="margin">
               <number>8</number>
              </property>
              <item alignment="Qt::AlignTop">
               <widget class="QLabel" name="trendingBoxHeader">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="text">
                 <string>Trending</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="trendingBoxImage">
                <property name="minimumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="beatsBox" native="true">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-width: 2px;
border-style: solid;
border-radius: 4px;
border-color: grey;
color: grey;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="margin">
               <number>8</number>
              </property>
              <item alignment="Qt::AlignTop">
               <widget class="QLabel" name="beatsBoxHeader">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="text">
                 <string>Beats</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="beatsBoxImage">
                <property name="minimumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="gmusicBox" native="true">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>79</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-width: 2px;
border-style: solid;
border-radius: 4px;
border-color: grey;
color: grey;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="margin">
               <number>8</number>
              </property>
              <item alignment="Qt::AlignTop">
               <widget class="QLabel" name="gmusicBoxHeader">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="text">
                 <string>GMusic</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="gmusicBoxImage">
                <property name="minimumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="networkingBox" native="true">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>79</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-width: 2px;
border-style: solid;
border-radius: 4px;
border-color: grey;
color: grey;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="margin">
               <number>8</number>
              </property>
              <item alignment="Qt::AlignTop">
               <widget class="QLabel" name="networkingBoxHeader">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="text">
                 <string>IPv6+</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="networkingBoxImage">
                <property name="minimumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="designBox" native="true">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>79</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-width: 2px;
border-style: solid;
border-radius: 4px;
border-color: grey;
color: grey;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="margin">
               <number>8</number>
              </property>
              <item alignment="Qt::AlignTop">
               <widget class="QLabel" name="designBoxHeader">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="text">
                 <string>Design</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="designBoxImage">
                <property name="minimumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item alignment="Qt::AlignTop">
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <widget class="QWidget" name="inboxPage">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Some Text about what can be done with the inbox Feature</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Display an Image about inbox here</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="urlLookupPage"/>
          <widget class="QWidget" name="trendingPage"/>
          <widget class="QWidget" name="beatsPage"/>
          <widget class="QWidget" name="gmusicPage"/>
          <widget class="QWidget" name="networkingPage"/>
          <widget class="QWidget" name="designPage"/>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="lineAbove">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="lineBelow">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
